modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -329,6 +329,8 @@ const RichText = ({
 									: ''
 							}
 							editorConfig={editorConfig}
+							helpMessage={otherProps.tip}
+							label={label}
```

Comment:
As this is just modifying standard config down the line, can we simplify solution and just something like this here:
```js
editorConfig={
    applicationTitle: label && `${label}, ${otherProps.tip}`,
    ...editorConfig,
}
```